src/main/java/com/easypost/service/EasyPostClient.java

```diff
@@ -6,11 +6,11 @@
 public class EasyPostClient {
     private final int connectTimeoutMilliseconds;
     private final int readTimeoutMilliseconds;
-    private final String apiKey;
+    private final String inputApiKey; // API key all EasyPost API requests
```

Comment:
Um... I don't think this was intended?

Reply:
If we rename the `apiKeys` to `apiKey`, then we need to rename the existing `apiKey` to something else

Reply:
ooooh... in some of our libs the API key API calls are in the User service/model... could be an option. That's how it is in PHP